openrtb_ext/bidders_test.go

```diff
@@ -114,6 +114,7 @@ func TestIsBidderNameReserved(t *testing.T) {
 		{"skADN", true},
 		{"SKADN", true},
 		{"notreserved", false},
+		{"tid", true},
```

Comment:
Please add a test case with mixed case for "tid" and for upper case to test the `EqualFold` method was used for the comparison. Please also move above the "notreserved" case to leave that last for readability (the only case which expected false).

Reply:
done